docs/howto/configure/auth-management.md

```diff
@@ -166,3 +166,59 @@ Presently, this involves a few more manual steps than the `auth0` setup describe
     ```
 
 4. Run the deployer as normal to apply the config.
+
+# CILogon
+
+For communities that require authenticating users against their campus identity providers, [CILogon](https://www.cilogon.org) can be used. CILogon authentication is managed through [auth0](https://auth0.com), similar with how the Google and GitHub authentication is setup.
+
+```{note}
+  The `USERNAME_KEY` used for the CILogon login is the email, so the JupyterHub username will be set to the email address associated with the CILogon connection and not the CILogon user_id!
```

Comment:
```suggestion
The JupyterHub username will be the email address that users provide when authenticating in CILogon connection. It will not be the CILogon `user_id`! This is because the `USERNAME_KEY` used for the CILogon login is the email address.
```